tests/commands/test_new.py

```diff
@@ -56,12 +87,11 @@ def test_folder_creation_with_new(repo: Repo, directory: str) -> None:
     Tests that folders are created when a sub-folder already exist, too.
```

Comment:
Commenting here, because GitHub won't let me comment higher... `overlap` is unused by this test and can be removed.

```
@pytest.mark.repo_dirs('overlap')
```

Reply:
It is used, because the folder `overlap` is the only case in which then a subfolder gets created, were a directory already exists, i.e. what is meant with the docstring "Tests that folders are created when a sub-folder already exist, too."

However I can add a comment to make this more obvious.